doc/user_guide/installing.rst

```diff
@@ -28,9 +29,9 @@ into an existing environment. The easiest way to do this is using conda/mamba.
    your url from ``http://localhost:8888/lab`` to ``http://localhost:8888/notebooks`` and use the
    classic notebook instead. There are less features but it is more stable in my experience.
 
-```
+
```

Comment:
I believe the best syntax here is to use double colons, as in "in my experience::". See https://raw.githubusercontent.com/pyxem/kikuchipy/develop/doc/user/installation.rst.

Reply:
Good suggestion.  I've made the change to `` ::``